tests/utils/json/tests.py

```diff
@@ -31,4 +33,12 @@ def __repr__(self):
                 return 'Unknown object'
 
         obj = Unknown()
-        assert json.dumps(obj) == '"Unknown object"'
```

Comment:
no mistake just github displaying fact my file has a newline character at the end of it in a weird way. View the file and this line is there.